nautobot/extras/tests/test_api.py

```diff
@@ -2236,6 +2238,84 @@ def test_dry_run_not_supported(self):
         self.assertHttpStatus(response, status.HTTP_405_METHOD_NOT_ALLOWED)
 
 
+class LogEntryTest(
+    APIViewTestCases.GetObjectViewTestCase,
+    APIViewTestCases.ListObjectsViewTestCase,
+    APIViewTestCases.NotesURLViewTestCase,
```

Comment:
Seems weird for this model to include a notes URL?

Reply:
Yeah good point.